short_tests/eddy_neknek/SIZE

```diff
@@ -0,0 +1,40 @@
+c
+c     Include file to dimension static arrays
+c     and to set some hardwired run-time parameters
+c
+      integer ldim,lx1,lxd,lx2,lx1m,lelg,lp,lelt,ldimt
+      integer ax1,ax2,lpx1,lpx2,lpelt,lbx1,lbx2,lbelt
+      integer toteq
+      integer lcvx1,lcvelt
+      integer lelx,lely,lelz,mxprev,lgmres,lorder,lhis
+      integer maxobj,maxmbr,lpert,nsessmax,nmaxl,nfldmax,nmaxcom
```

Comment:
SIZE.inc already declares nfldmax, nmaxcomm.  Declaring them here might lead to multiple declaration errors?